data/race.json

```diff
@@ -1,7 +1,466 @@
+		],
+		"height": [157, 195],
+		"weight": -110,
+		"ATR": [0, 0, 1, 0, 0, 1, 0],
+		"LP": 9,
+		"SP": 12,
+		"MR": -5,
+		"auto perk_harm": [],
+		"fit perk_harm": [
+			["Ausdauernd", 1],
+			["Entfernungssinn", 1],
+			["Gefahreninstinkt", 1],
+			["Innerer Kompass", 1],
+			["Kälteresistenz", 1],
+			["Richtungssinn", 1],
+			["Wolfskind", 1],
+			["Zäher Hund", 1],
+			["Hitzeempfindlich", 1]
+		],
+		"unfit perk_harm": [
+			"Herausragende Balance",
+			"Nahrungsrestriktion",
+			"Hitzeresistenz",
+			"Kälteempfindlich",
+			"Platzangst"
+		],
+		"fit culture": [
+			"Nivesenstaemme",
+			"Nuanaae·Lie"
+		],
+		"pos culture": [
+			"Mittellaendische Staedte",
+			"Mittellaendische Landbevoelkerung",
+			"Bornland",
+			"Svellttal",
+			"Thorwal",
+			"Gjalskerland",
+			"Fjarninger",
+			"Norbardensippen"
+		],
+		"talents": [
+			["Schleichen", 1],
+			["Selbstbeherrschung", 1],
+			["Orientierung", 1]
+		]
+	},
+	{
+		"name": "NORBARDEN",
+		"GP": 3,
+		"hair color": [
+			[17, "schwarz"],
+			[19, "kupferrot"],
+			[20, "blond"]
+		],
+		"eye color": [
+			[3, "schwarz"],
+			[13, "dunkelbraun"],
+			[17, "braun"],
+			[19, "gruen"],
+			[20, "blau"]
+		],
+		"height": [160, 198],
+		"weight": -100,
+		"ATR": [0, 0, 0, 1, 0, 0, 0],
+		"LP": 11,
+		"SP": 10,
+		"MR": -4,
+		"auto perk_harm": [],
+		"fit perk_harm": [
+			["Gefahreninstinkt", 1],
+			["Gutes Gedaechtnis", 1],
+			["Innerer Kompass", 1],
+			["Kälteresistenz", 1],
+			["Richtungssinn", 1], 
+			["Zäher Hund", 1]
+		],
+		"unfit perk_harm": [
+			"Herausragende Balance",
+			"Hitzeresistenz",
+			"Nahrungsrestriktion",
+			"Platzangst"
+		],
+		"fit culture": [
+			"Norbardensippe"
+		],
+		"pos culture": [
+			"Mittellaendische Staedte",
+			"Mittellaendische Landbevoelkerung",
+			"Bornland",
+			"Thorwal",
+			"Svellttal",
+			"Nivesenstaemme",
+			"Gjalskerland"
+		],
+		"talents": [
+			["Orientierung", 1],
+			["Rechnen", 1]
+		]
+	},
+	{
+		"name": "LOOOL",
+		"GP": 3,
+		"hair color": [
+			[17, "schwarz"],
+			[19, "kupferrot"],
+			[20, "blond"]
+		],
+		"eye color": [
+			[3, "schwarz"],
+			[13, "dunkelbraun"],
+			[17, "braun"],
+			[19, "gruen"],
+			[20, "blau"]
+		],
+		"height": [160, 198],
+		"weight": -100,
+		"ATR": [0, 0, 0, 1, 0, 0, 0],
+		"LP": 11,
+		"SP": 10,
+		"MR": -4,
+		"auto perk_harm": [],
+		"fit perk_harm": [
+			["Gefahreninstinkt", 1],
+			["Gutes Gedaechtnis", 1],
+			["Innerer Kompass", 1],
+			["Kälteresistenz", 1],
+			["Richtungssinn", 1], 
+			["Zäher Hund", 1]
+		],
+		"unfit perk_harm": [
+			"Herausragende Balance",
+			"Hitzeresistenz",
+			"Nahrungsrestriktion",
+			"Platzangst"
+		],
+		"fit culture": [
+			"Norbardensippe"
+		],
+		"pos culture": [
+			"Mittellaendische Staedte",
+			"Mittellaendische Landbevoelkerung",
+			"Bornland",
+			"Thorwal",
+			"Svellttal",
+			"Nivesenstaemme",
+			"Gjalskerland"
+		],
+		"talents": [
+			["Orientierung", 1],
+			["Rechnen", 1]
+		]
 	}
-]}
+
+MU, KL, IN, CH, FF, GE, KO, KK
+	
+	
```

Comment:
what is the following text? the raw pulled data, or data of races still to be added?